### DISCUSSION TITLE
Ideas for Collaboration

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [WebXDAO/WebXDAO.github.io#531] [Ideas]
We're discussing ideas or projects that can be done in about 20 days and would be useful for real people.

Here are some ideas:

1. Make a website that shows all open-source events. This way, people won't have to search through multiple blogs and websites.

2. Create a tool where you can check if a username is available on all developer platforms. It's great for personal branding.

3. Build a place where people can submit their AI tools. We can then show these tools on the website. There are already more than 10,000 AI tools out there, so having them in one place would be handy.

4. Make a simple website where WebXDAO members can find job opportunities.

> As a community, if we can make cool shit. It would be great.

Small Project -> Big Impact

**Comment by vinzvinci** [comment]
These are great ideas! On number 1, we're already working on it, as we mentioned earlier on Discord, but it will be a long journey for us. We have separated the org. for it, you can check my dm on discord.

I like the 2, 3, 4 to continue here in WebX DAO.

**Comment by mkubdev** [comment]
> Small Project -> Big Impact

100% agree.

Hi Anmol! This is great ideas! I would love to collaborate on the project 2 as it's something quick to develop indeed. 🚀
See you soon for project 1 :)
